minhton/src/core/routing_information.cpp

```diff
@@ -208,21 +210,22 @@ void RoutingInformation::setChild(const minhton::NodeInfo &child, uint16_t posit
     throw std::out_of_range("Position not in range");
   }
 
-  if (this->children_[position].getPeerInfo() != child.getPeerInfo()) {
+  if (this->children_[position].getLogicalNodeInfo() != child.getLogicalNodeInfo()) {
     throw std::invalid_argument(
-        "The NodeInfo needs to have the correct PeerInfo position in the tree.");
+        "The NodeInfo needs to have the correct LogicalNodeInfo position in the tree.");
   }
 
   if (this->getFanout() != child.getFanout()) {
-    throw std::invalid_argument("The Fanout needs to be set properly in the PeerInfo object");
+    throw std::invalid_argument(
+        "The Fanout needs to be set properly in the LogicalNodeInfo object");
   }
 
   if (!child.isInitialized()) {
     throw std::invalid_argument("Child must be initialized / must exist");
   }
 
   // if there is a difference in network info
```

Comment:
Could argue that this is not entirely accurate anymore, but the comment is kind of unnecessary anyway.

```suggestion

```

Reply:
Comment removed.